src/execution/StepExecutionManager.ts

```diff
@@ -76,35 +70,38 @@ export class StepExecutionManager {
 
     if (process.status !== 'DONE') {
       // TODO: update @lifi/types repo for these changes [DO NOT MERGE IF YOU SEE THIS]
-      if (isSafeSigner && step.execution.status === 'SAFE_SIGNING_PENDING') {
-        console.log('Alrighty coming here')
-
+      if (
+        isSafeSigner &&
+        step.execution.status === 'MULTISIG_SIGNING_PENDING'
+      ) {
         const multiSigProcess = step.execution.process.find(
-          (p) => p.status === 'SAFE_SIGNING_PENDING'
+          (p) => p.status === 'MULTISIG_SIGNING_PENDING'
```

Comment:
We agreed not to have a separate status for execution/process and reuse ACTION_REQUIRED one 👀

Reply:
I thought we did PENDING :/

Reply:
> I thought we did PENDING :/

Yes, PENDING/ACTION_REQUIRED, but without an introduction of a new one.